Assets/css/customer.css

```diff
@@ -0,0 +1,303 @@
+*{
+    margin: 0;
+    padding: 0;
+}
+
+body{
+    overflow-y: scroll;
+    font-family: 'Overpass', sans-serif;
+}
+
+#root{
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+}
+
+
+/* Header */
+header{
+    position: fixed;
+}
+
+.header-div{
+    width: 1490px;
```

Comment:
insted of giving the `header-div` fixed width, give it `100vw` to make it responsive on all screens.

Reply:
resolved